pkg/apis/flagger/v1beta1/canary.go

```diff
@@ -394,6 +394,9 @@ type CanaryWebhook struct {
 	// Metadata (key-value pairs) for this webhook
 	// +optional
 	Metadata *map[string]string `json:"metadata,omitempty"`
+
+	// Number of retries for this webhook
+	Retries int `json:"retries,omitempty"`
```

Comment:
```suggestion
	// Number of retries for this webhook
	// +optional
	Retries int `json:"retries,omitempty"`
```

Reply:
After you run the generator, add the retries filed to `artifacts/flagger/crd.yaml` and run `make crd`. This will update the CRDs in all places.